### PR TITLE
Add nginx to docker and make application available publicly.

### PR DESCRIPTION
**Pull request by simonseyock**

### CODE REVIEW

File: docker/docker-compose.yaml [hunk 1]
```diff
@@ -1,5 +1,13 @@
-version: '3.7'
+version: '3.8'
 services:
+  munimap-nginx:
+    image: nginx:alpine
+    ports:
+      - 80:80
```
Comment (Unraveler) [inline docker/docker-compose.yaml:6]: should port 443 also be opened to handle https or is there no need for it?

File: docker/nginx/default.conf [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+#         access_log /var/log/nginx/svgserver.access.log;
+#         error_log /var/log/nginx/svgserver.error.log;
+#
+#         proxy_pass http://127.0.0.1:xxxx;
+#         proxy_set_header Host $http_host;
+#         proxy_set_header X-Script-Name $http_x_script_name/svgserver;
+#         proxy_set_header X-Forwarded-Proto $scheme;
+#         proxy_read_timeout 300;
+#     }
+}
```
Comment (Unraveler) [inline docker/nginx/default.conf:69]: I see now there is no ssl certificates involved. So you can forget the previous comment

**Review by Unraveler — APPROVED** [review]
Looks good to me

File: docker/docker-compose.yaml [hunk 3]
```diff
@@ -1,5 +1,13 @@
-version: '3.7'
+version: '3.8'
 services:
+  munimap-nginx:
+    image: nginx:alpine
```
Comment (dnlkoch) [inline docker/docker-compose.yaml:4]: Please specify the image tag.

**Pull request merged**